src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -78,13 +79,18 @@ public PersonCard(Person person, int displayedIndex) {
                 tags.getChildren().add(label);
             });
         person.getNonEmergencyTags().stream()
-                .sorted(Comparator.comparing(tag -> tag.tagName))
-                .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
-        uniqueId.setText(person.getId().map(x -> Integer.toString(x)).orElse("NONE"));
+            .sorted(Comparator.comparing(tag -> tag.tagName))
```

Comment:
Java Coding Standard violation -- Indentation for wrapped lines should be 8 spaces